python/paddle/v2/reader/decorator.py

```diff
@@ -224,3 +226,74 @@ def firstn_reader():
             yield item
 
     return firstn_reader
+
+
+class XmapEndSignal():
+    pass
+
+
+def xmap(mapper, reader, process_num, buffer_size):
+    """
+    Use multiprocess to map samples from reader by a mapper defined by user.
```

Comment:
I vaguely remember that @helinwang had a function which uses multiprocess to accelerate loading.  Could @helinwang please confirm?